webapp/channels/src/i18n/en.json

```diff
@@ -5893,5 +5893,14 @@
   "workspace_limits.teams_limit_reached.view_upgrade_options": "View upgrade options",
   "workspace_limits.upgrade": "Upgrade to avoid {planName} data limits",
   "workspace_limits.upgrade_reasons.free": "{planName} is restricted to {messagesLimit} message history and {storageLimit} file storage. You can delete items to free up space or upgrade to a paid plan.",
-  "yourcomputer": "Your computer"
+  "yourcomputer": "Your computer",
+  "channel_info_rhs.menu.files.filter.file_menu" : "file menu",
```

Comment:
I think this should be moved up as it is expected to be in alphanumeric order. It's likely the reason why it is failing the test

Reply:
will fix them